src/renderer/services/docker/chain.ts

```diff
@@ -0,0 +1,42 @@
+import {Container} from "./container";
+
+type LogType = "info" | "error";
+type LogCallbackFunc = (type: LogType, message: string) => void;
+
+export enum SupportedNetworks {
+    PRYSM = "Prysm",
+}
+
+export class BeaconChain extends Container {
+    public static instance: BeaconChain;
```

Comment:
Hmm i think better solution is to have singleton DockerContainers where each "Container" class could register itself.
In that way you could generate multiple BeaconChain containers like for local and public testnet at the same time